File: go-sdk/pkg/models/operations/getcommentbyid.go
package operations

type GetCommentByIDPathParams struct {
	ID int64 `pathParam:"style=simple,explode=false,name=id"`
}

type GetCommentByIDRequest struct {
	PathParams GetCommentByIDPathParams
}

type GetCommentByIDResponse struct {
	ContentType string
	StatusCode  int64
}

File: ts-sdk/src/sdk/models/operations/getcommentbyid.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



export class GetCommentByIdPathParams extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=id" })
  id: number;
}


export class GetCommentByIdRequest extends SpeakeasyBase {
  @SpeakeasyMetadata()
  pathParams: GetCommentByIdPathParams;
}


export class GetCommentByIdResponse extends SpeakeasyBase {
  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  statusCode: number;
}
